platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/gossip/sync/SingleNodeSyncGossip.java

```diff
@@ -0,0 +1,255 @@
+     * @param appVersion                the version of the app
+     * @param shadowGraph               contains non-ancient events
+     * @param intakeQueue               the event intake queue
+     * @param freezeManager             handles freezes
+     * @param startUpEventFrozenManager prevents event creation during startup
+     * @param swirldStateManager        manages the mutable state
+     * @param stateManagementComponent  manages the lifecycle of the state
+     * @param eventIntakeLambda         a method that is called when something needs to be added to the event intake
+     *                                  queue
+     * @param eventObserverDispatcher   the object used to wire event intake
+     * @param eventMapper               a data structure used to track the most recent event from each node
+     * @param eventIntakeMetrics        metrics for event intake
+     * @param updatePlatformStatus      a method that updates the platform status, when called
+     * @param loadReconnectState        a method that should be called when a state from reconnect is obtained
+     */
+    public SingleNodeSyncGossip(
+            @NonNull PlatformContext platformContext,
+            @NonNull ThreadManager threadManager,
+            @NonNull Crypto crypto,
+            @NonNull AddressBook addressBook,
+            @NonNull NodeId selfId,
+            @NonNull SoftwareVersion appVersion,
+            @NonNull final ShadowGraph shadowGraph,
+            @NonNull final QueueThread<EventIntakeTask> intakeQueue,
+            @NonNull final FreezeManager freezeManager,
+            @NonNull final StartUpEventFrozenManager startUpEventFrozenManager,
+            @NonNull final SwirldStateManager swirldStateManager,
+            @NonNull final StateManagementComponent stateManagementComponent,
+            @NonNull final InterruptableConsumer<EventIntakeTask> eventIntakeLambda,
+            @NonNull final EventObserverDispatcher eventObserverDispatcher,
+            @NonNull final EventMapper eventMapper,
+            @NonNull final EventIntakeMetrics eventIntakeMetrics,
+            @NonNull final Runnable updatePlatformStatus,
+            @NonNull final Consumer<SignedState> loadReconnectState) {
+        super(
+                platformContext,
+                threadManager,
+                crypto,
+                addressBook,
+                selfId,
+                appVersion,
+                intakeQueue,
+                freezeManager,
+                startUpEventFrozenManager,
+                swirldStateManager,
+                stateManagementComponent,
+                eventMapper,
+                eventIntakeMetrics,
+                eventObserverDispatcher,
+                updatePlatformStatus,
+                loadReconnectState);
+
+        this.eventIntakeLambda = Objects.requireNonNull(eventIntakeLambda);
+
+        final ParallelExecutor shadowgraphExecutor = PlatformConstructor.parallelExecutor(threadManager);
```

Comment:
I think you can remove this entirely - this `ParallelExecutor` doesn't serve any purpose that I can see